packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -400,7 +400,7 @@ const Acls = decorate([
     computed: {
       rawAcls: (_, { acls, vm }) => filter(acls, { object: vm }),
       resolvedAcls: ({ rawAcls }, { users, groups }) => {
-        if (users === undefined && groups === undefined) {
+        if (users === undefined || groups === undefined) {
```

Comment:
For example, If `users` is `undefined` and `groups` is not, in this case the `resolvedAcls` will be `[]` [but it may not be empty](https://github.com/vatesfr/xen-orchestra/pull/4578/files#diff-8eaaeaccc6fc24d6a56b6f26c39c0cb3R408).

Reply:
In which case `users` is `undefined`?

Reply:
As discussed, `users` and `groups` are not `undefined` if they are fetched.
[Please, remove this test also](https://github.com/vatesfr/xen-orchestra/pull/4578/files#diff-8eaaeaccc6fc24d6a56b6f26c39c0cb3R409).